tests/bin/install-phpunit-tests-dependencies.sh

```diff
@@ -60,12 +62,15 @@ install_wp() {
 }
 
 install_woocommerce() {
-    cd $TRAVIS_BUILD_DIR
-    cd ..
-    download https://codeload.github.com/woocommerce/woocommerce/tar.gz/3.2.5 woocommerce.tar.gz
-    mkdir woocommerce
-    tar --strip-components=1 -zxmf woocommerce.tar.gz -C woocommerce
-    cd -
+	cd $TRAVIS_BUILD_DIR
+	cd ..
+	git clone https://github.com/woocommerce/woocommerce.git
+	cd woocommerce
+
+	git fetch && git fetch --tags
+	git checkout 4.3.1
```

Comment:
There is no need to clone whole repo. Specific tag can be fetched like this:
```suggestion
	git clone --depth=1 --branch ${WC_VERSION-4.3.1} https://github.com/woocommerce/woocommerce.git
	cd woocommerce
```

Reply:
Yes, much better! Applied in https://github.com/woocommerce/woocommerce-gateway-amazon-pay/pull/57/commits/aab259e6e941e1efaccf21aaed8b988e53066198.